rebar.config.erl

```diff
@@ -255,13 +255,13 @@ relx_apps(ReleaseType) ->
     , inets
     , compiler
     , runtime_tools
-    , {emqx, load} % started by emqx_machine
-    , {emqx_conf, load}
     , emqx_machine
-    , {mnesia, load}
-    , {ekka, load}
-    , {emqx_plugin_libs, load}
-    , {esasl, load}
+    , emqx_conf
+    , emqx
+    , mnesia
```

Comment:
Please don't change ekka and mnesia from load only to a start. This will break cluster joining procedure, since ekka/mria have to manage the schema before mnesia starts. So they have to have the full control of starting mnesia.

Reply:
That's the reason I move `mnesia` and `ekka` behind `emqx_machine`.
`emqx_machine` will start early by master supervisor than `ekka` when we start the application, and ekka will start by `emqx_machine` https://github.com/emqx/emqx/blob/master/apps/emqx_machine/src/emqx_machine.erl#L35
would it be an issue?